packages/lit-html/src/directives/class-map.ts

```diff
@@ -12,7 +12,7 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-import {AttributePart, noChange, PartInfo} from '../lit-html.js';
+import {AttributePart, DirectiveParameters, noChange, PartInfo} from '../lit-html.js';
```

Comment:
Nit: `DirectiveParameters` is also exported from `directive.js`, might be nicer to see all the directive stuff imported from there. But nice that you cleaned this up.

Reply:
I'm going to do another pass on where the directive related interfaces are exported from...